src/backend/main.ts

```diff
@@ -192,6 +193,16 @@ async function initializeWindow(): Promise<BrowserWindow> {
     }
   }, 2500)
 
+  if (!isWindows) {
```

Comment:
maybe we can do this also only when opening heroic directly? Now if you open a game from a shortcut it will update the wine versions in the background, and if you open heroic later it will do that again

I think adding an extra `&& !isCLINoGui` condition in the if is enough?